Application/EdFi.Ods.SandboxAdmin/Views/Client/Index.cshtml

```diff
@@ -104,15 +104,23 @@
                 <fieldset>
                     <p>Creating new sandboxes -- especially populated sandboxes may take a few minutes.</p>
                     <div class="control-group">
-                        <label class="control-label" for="txtName">Sandbox Name:</label>
+                        <label class="control-label" for="txtSandboxName">Sandbox Name:</label>
                         <div class="controls">
-                            <input id="txtName" type="text" data-bind="value: applicationName, valueUpdate: 'keyup'" class="form-control" placeholder="Name of Sandbox" />
+                            <input id="txtSandboxName" type="text" data-bind="value: sandboxName" class="form-control" placeholder="Name of Sandbox" />
                         </div>
                     </div>
                     <div class="control-group">
-                        <label class="control-label">Application:</label>
                         <div class="controls">
-                            <select data-bind="options: applicationList, value:selectedApplication, optionsText: 'Name', valueUpdate: 'keyup' " class="dropdown"></select>
+                            <div class="dropdown">
+                                <label class="control-label" for="txtApplication">Application:</label>
+                                <a href="#" class="nav-link dropdown-toggle" id="application-dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    Select Application
```

Comment:
Selected application is not displayed
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/43448209/c171bf93-923d-4ff7-bc5d-1409e998276e)

Reply:
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/34613894/80d8eb6b-750b-4c6f-bd8b-34be63801450)
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/34613894/768e1188-d05e-4bdf-9ec0-66570a4cd478)